src/config.zig

```diff
@@ -296,6 +296,7 @@ pub const configs = struct {
     pub const fuzz_min = config: {
         var base = test_min;
         base.process.storage_size_limit_max = 1 * 1024 * 1024 * 1024;
+        base.cluster.lsm_scans_max = 12;
```

Comment:
Ouch, I didn't realize we are adjusting fuzz_min here, I have just removed it on main :crying_cat_face:

Reply:
We can live with the production value, the only problem is that right now it's too low to stress any meaningful condition; However it soon will be increased soon by another PR.

I just need to adjust the code to consider the minimum between scans_max and the fields count.